docs_user/modules/proc_preparing-block-storage-service-by-customizing-configuration.adoc

```diff
@@ -13,38 +13,18 @@ The high level explanation of the tailor-made approach is:
 endif::[]
 
 .Procedure
-. Determine what part of the configuration is generic for all the {block_storage_first_ref}
-services and remove anything that would change when deployed in {OpenShift}, like
-the `connection` in the `[dabase]` section, the `transport_url` and `log_dir` in
-`[DEFAULT]`, the whole `[coordination]` and `[barbican]` sections.  This
-configuration goes into the `customServiceConfig` (or a `Secret` and then used
-in `customServiceConfigSecrets`) at the `cinder: template:` level.
-. Determine if there's any scheduler specific configuration and add it to the
+. Determine what part of the configuration is generic for all the Block Storage services and remove anything that would change when deployed in {OpenShift} ({OpenShiftShort}), such as the `connection` in the `[dabase]` section, the `transport_url` and `log_dir` in the `[DEFAULT]` sections, and the whole `[coordination]` and `[barbican]` sections. This configuration goes into the `customServiceConfig`, or a `Secret` and then used in `customServiceConfigSecrets`, at the `cinder: template:` level.
```

Comment:
`[dabase]` ==> `[database]`

I think the last sentence should be updated to make it a bit easier to read:
`The generic configuration that has not been removed goes into the`...